__tests__/app.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+
+const {
+  categoryData,
+  commentData,
+  reviewData,
+  userData,
+} = require("../db/data/test-data");
+
+beforeEach(() =>
+  seed({
+    categoryData,
+    commentData,
+    reviewData,
+    userData,
+  })
+);
+
+afterAll(() => db.end());
+
+describe("NC-Games app", () => {
+  describe("GET /api/categories", () => {
+    test("Responds with status 200 and an array of categories with length 4", () => {
```

Comment:
You can get rid of this test block because you are already testing the length of the array in the next test block.